### PR TITLE
feat(activity): add ActivityAdapter interface

### PR DESCRIPTION
**Pull request by lalli-flores**

### CODE REVIEW

File: src/adapters/ActivityAdapter.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * This is a base class that defines the interface that maps activity data.
+ * Developers that want to extend `ActivityAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class ActivityAdapter {
+  /**
+   * Returns an observable that emits message data of the given ID.
+   *
+   * @param {String} id - ID of the message to get
+   * @returns {Observable<Activity>}
```
Comment (adamweeks) [inline src/adapters/ActivityAdapter.js]: Are we going to define the shape of the `<Activity>` object in the jsdoc as well?

Comment (lalli-flores) [reply src/adapters/ActivityAdapter.js]: Yes, that's a good idea!

File: src/adapters/ActivityAdapter.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import {throwError} from 'rxjs';
+
+/**
+ * This is a base class that defines the interface that maps activity data.
+ * Developers that want to extend `ActivityAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class ActivityAdapter {
```
Comment (adamweeks) [inline src/adapters/ActivityAdapter.js]: Don't forget to extend from the base WebexAdapter class!

**Pull request merged**